perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/PerunFormItemsGeneratorImpl.java

```diff
@@ -96,16 +226,6 @@ private PerunFormItem generatePerunFormItem(ApplicationFormItemData data) {
 				return new Checkbox(form, data, lang);
 			case RADIO:
 				return new Radiobox(form, data, lang);
-			case FROM_FEDERATION_SHOW:
```

Comment:
Above case for `USERNAME` ensures, that its not editable when prefilled. Its then not corresponding with the item settings (disabled when - never). We should probably update DB change-log in core, so that login items are disabled if prefilled. Or generally discuss this use-case in more detail.

Reply:
Ok, I would keep it that way for now, and we can discuss future improvements.